lib/components/blockquote.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/forbid-prop-types */
+
+import React from 'react';
+
+const setup = ({ blockList }) => {
+  const Blockquote = ({ items, pullQuote = false }) =>
+    <blockquote className={pullQuote ? 'q' : null}>{blockList(items)}</blockquote>;
```

Comment:
A blockquote element can all elements in the flow content category.  See [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote). The article-json structure is setup to be able to handle this, so we need to also be able to render that here.